styles/_header.less

```diff
@@ -34,6 +34,7 @@
 .site-header__mediumScreenNav a:hover,
 .site-header__smallScreenNavContainer a,
 .activeNavLink {
+  color: @badgerRedOnWhite;
   border-bottom: 2px solid red;
```

Comment:
This is where I meant to leave my last comment - change to border-bottom: 2px solid @badgerRedOnWhite

This really should be the last change (honestly)